apps/spruce/src/pages/version/NameChangeModal/index.tsx

```diff
@@ -53,18 +53,21 @@ export const NameChangeModal: React.FC<NameChangeModalProps> = ({
         <Icon glyph="Edit" />
       </StyledIconButton>
       <ConfirmationModal
-        buttonText="Confirm"
-        onCancel={() => setIsOpen(false)}
-        onConfirm={() => {
-          updateDescription({
-            // @ts-expect-error: FIXME. This comment was added by an automated script.
-            variables: { patchId, description: newPatchName },
-          });
+        cancelButtonProps={{
+          onClick: () => setIsOpen(false),
+        }}
+        confirmButtonProps={{
+          children: "Confirm",
+          disabled:
+            newPatchName === originalPatchName || hasFormError || loading,
+          onClick: () => {
+            updateDescription({
+              // @ts-expect-error: FIXME. This comment was added by an automated script.
```

Comment:
I know this was here before but would it be a big lift to fix this expect-error since we are already updating this part of the codebase?